examples/bridge-app/linux/bridged-actions-stub.cpp

```diff
@@ -20,12 +20,12 @@
 #include <app-common/zap-generated/ids/Clusters.h>
 #include <app/AttributeAccessInterface.h>
 #include <app/AttributeAccessInterfaceRegistry.h>
+#include <app/clusters/actions-server/actions-server.h>
 #include <app/util/attribute-storage.h>
+#include <bridged-actions-stub.h>
```

Comment:
nit: it would be better to keep non-SDK (the library part) includes in a separate block. The best way to do it would be to define include groups in clang-format, so probably that's something for totally separate PR.